src/signals/incident/definitions/wizard-step-3-contact.js

```diff
@@ -31,7 +31,7 @@ export default {
       phone: {
         meta: {
           // https://bytes.grubhub.com/disabling-safari-autofill-for-a-single-line-address-input-b83137b5b1c7
-          autoComplete: 'tel',
+          autoComplete: 'search_tel',
```

Comment:
changed this back, after discussions with po, cause autocomplete is currently not cooperating with react's controlled inputs.